pkg/console/starter/starter.go

```diff
@@ -152,6 +154,15 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		resourceSyncer,
 	)
 
+	service.NewServiceSyncController(
```

Comment:
We will likely have a bunch of controllers starting here, just all sharing the informers, clients, etc that we have already created.